app/lib/actions/ui.js

```diff
@@ -1,6 +1,8 @@
+import { php_escapeshellcmd } from 'php-escape-shell';
```

Comment:
I know, I follow the code style standards too: the author of php-shell-escape doesn't.
Ok, I'll alias it, I just explained why I kept the original name.